lib/workers/repository/error-config.js

```diff
@@ -32,7 +32,9 @@ async function raiseConfigWarningIssue(config, error) {
   } else {
     const res = await platform.ensureIssue(
       `Action Required: Fix ${appName} Configuration`,
-      body
+      body,
+      false,
+      true
```

Comment:
Let's define these as variables `once` and `reopen` before passing them into `ensureIssue`. That way it's more understandable.